tests/snapshots/tests.contrib.rq.test_rq.test_queue_pin_service.json

```diff
@@ -32,6 +33,7 @@
        "parent_id": 1,
        "type": "worker",
        "meta": {
+         "_dd.p.dm": "-0",
```

Comment:
For some reason `-0` isn't evaluating here and is ending up with a `'_dd.propagation_error': 'decoding_error'` instead. With the error message `failed to decode _dd.p.dm: '-0'`. However I'm confused by this, as there are many `'-0'` values that seem not to have this issue. I tried testing with `TRACE_TAG_RE.match(value)` but it's definitely evaluating to None for some reason...

Reply:
interesting, I would expect this to match the same snapshot as `test_sync_worker` (but the service names different because of the pin)

Reply:
I figured it out, needed to change the regex expression to not have the service hash in there.